exporter/exporter_test.go

```diff
@@ -59,40 +60,57 @@ func getTestExporterWithOptions(opt Options) *Exporter {
 	return e
 }
 
-func setupDBKeys(t *testing.T, uri string) error {
-	c, err := redis.DialURL(uri)
+func createClusterClient(uri string) (redis.Conn, error) {
```

Comment:
Why a repeat all the logic here and not use `connectToRedisCluster()` instead? 

All you need to do is set `redisAddr`, do something like this:

```
e := Exporter{redisAddr: uri}
c, err :=  e.connectToRedisCluster()
...
```